java/pom.xml

```
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project  xmlns="http://maven.apache.org/POM/4.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>ai.rapids.kvikio</groupId>
+  <artifactId>cufile</artifactId>
+  <version>24.12.0-SNAPSHOT</version>
+
+  <name>cufile</name>
+  <description>
+      This project provides Java bindings for the GPUDirect Storage cufile library, enabling the GPU to load and
+      save large amounts of data to and from persistent storage. This is still a work in progress so some APIs may change.
+  </description>
+  <url>http://ai.rapids</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>21</maven.compiler.source>
+    <maven.compiler.target>21</maven.compiler.target>
```

Maybe the build failures are expecting Java 21, but only 11 is present?
```suggestion
    <maven.compiler.source>11</maven.compiler.source>
    <maven.compiler.target>11</maven.compiler.target>
```

---

Yes this problem I was able to fix, there is another linking issue with the libraries that I am running into now though. Let me push up my latest updates